GRDB/Core/Database+Schema.swift

```diff
@@ -365,8 +365,12 @@ extension Database {
                 throw DatabaseError(message: "no such table: \(tableName)")
             }
         }
+        // The "hidden" column magic numbers come from the SQLite source code.
+        // The values 2 and 3 refer to virtual and stored generated columns,
+        // respectively. Search for COLFLAG_VIRTUAL in
+        // https://www.sqlite.org/cgi/src/file?name=src/pragma.c&ci=fca8dc8b578f215a
```

Comment:
Thanks for this comment. Considering the focus on backward compatibility displayed by the authors of SQLite, it is pretty safe to rely on the source code when documentation is lacking. 👍